### PR TITLE
Update github workflows

### PR DESCRIPTION
**Pull request by svetomir**
Update checkout action version.
Specify Golang versions.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -11,15 +11,12 @@ jobs:
...
-
-      - name: Checkout
-        uses: actions/checkout@v1
-
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.x
-
+          go-version: '1.13'
```
Comment (janos) [inline .github/workflows/release.yaml]: No quotes needed.

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
 on: [push, pull_request]
-jobs:
 
+jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        go: ['1.13', '1.x']
```
Comment (janos) [inline .github/workflows/go.yml]: I think that here quotes are not needed. I've noticed that sometimes 1.13.x notation is used. Is 1.13 selecting the latest minor release, or the first one?

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -11,15 +11,12 @@ jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
-
-      - name: Checkout
-        uses: actions/checkout@v1
-
       - name: Set up Go
```
Comment (janos) [inline .github/workflows/release.yaml]: This step name should be updated to reflect the changes on go.yml.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (svetomir) [reply .github/workflows/release.yaml]: Ok, I removed the quotes.

File: .github/workflows/go.yml (hunk 2 above)
Comment (svetomir) [reply .github/workflows/go.yml]: I removed the quotes. It is selecting latest minor release.

File: .github/workflows/release.yaml (hunk 3 above)
Comment (svetomir) [reply .github/workflows/release.yaml]: I updated the name.

**Pull request merged**